<launch>
  <group ns="astraprosm">
    <node pkg="libuvc_camera" type="camera_node" name="left">
      <!-- Parameters used to find the camera -->
      <param name="vendor" value="0x2bc5"/>
      <param name="product" value="0x0501"/>
      <param name="serial" value="0"/>
      <!-- If the above parameters aren't unique, choose the first match: -->
      <param name="index" value="0"/>

      <!-- Image size and type -->
      <param name="width" value="640"/>
      <param name="height" value="480"/>
      <!-- choose whichever uncompressed format the camera supports: -->
      <param name="video_mode" value="yuyv"/> <!-- or yuyv/nv12/jpeg -->
      <param name="frame_rate" value="30"/>

      <param name="timestamp_method" value="start"/> <!-- start of frame -->
      <param name="camera_info_url" value="file:///home/cesare/.ros/camera_info/left_camera.yaml"/>

      <!-- <param name="auto_exposure" value="3"/> --> <!-- use aperture_priority auto exposure -->
      <!-- <param name="auto_white_balance" value="false"/> -->
    </node>
  </group>
</launch>
